{
  "_annotation": {
    "cve_id": "CVE-2024-53949",
    "generated_from": "https://raw.githubusercontent.com/anchore/cve-data-enrichment/main/data/anchore/2024/CVE-2024-53949.json",
    "reason": "Added CPE configurations because not yet analyzed by NVD."
  },
  "cve": {
    "configurations": [
      {
        "nodes": [
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:apache:superset:*:*:*:*:*:python:*:*",
                "matchCriteriaId": "51A03208-B4F6-5BD4-BFB2-F6803E3C427A",
                "versionEndExcluding": "4.1.0",
                "versionStartIncluding": "2.0.0",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          }
        ]
      }
    ]
  }
}